Sources/AardvarkMailUI/ARKEmailBugReporter.h

```diff
@@ -103,6 +103,6 @@ typedef void (^ARKEmailBugReporterCustomPromptCompletionBlock)(ARKEmailBugReport
 @property (nonatomic) BOOL attachesViewHierarchyDescription;
 
 /// Returns an attachment containing the log messages. Defaults to a plain text attachment containing each log message formatted using the bug reporter's `logFormatter`.
-- (nullable ARKBugReportAttachment *)attachmentForLogMessages:(nonnull NSArray<ARKLogMessage *> *)logMessages inLogStoreNamed:(nonnull NSString *)logStoreName;
+- (nullable ARKBugReportAttachment *)attachmentForLogMessages:(nonnull NSArray<ARKLogMessage *> *)logMessages inLogStoreNamed:(nonnull NSString *)logStoreName numberOfErrorsInHighlights:(NSInteger)numberOfErrorsInHighlights;
```

Comment:
This is a breaking change for AardvarkMailUI. We could potentially work around this, but I think we should go ahead and call this a breaking change, since there will be other breaking changes to follow (in making the log store be treated like any other attachment in the email bug reporter).